README.md

```diff
@@ -84,552 +84,284 @@ We conducted a comprehensive evaluation of Qwen2.5-Omni, which demonstrates stro
+| Qwen2-Audio       | 0.553    |
+| Qwen2.5-Omni-7B   | **0.570**|
+
+### VSC
+
+#### Dataset: VocalSound
+
+| Model           | Accuracy |
+|-----------------|----------|
+| CLAP            | 0.495    |
+| Pengi           | 0.604    |
+| Qwen-Audio      | 0.929    |
+| Qwen2-Audio     | **0.939**|
+| Qwen2.5-Omni-7B | **0.939**|
+
+### Music
+
+#### Dataset: GiantSteps Tempo
+
+| Model           | Score |
+|-----------------|-------|
+| Llark-7B        | 0.86  |
+| Qwen2.5-Omni-7B | **0.88** |
+
+#### Dataset: MusicCaps
+
+| Model             |         |         |         |          |          |         |
+|-------------------|---------|---------|---------|----------|----------|---------|
+| LP-MusicCaps      | 0.291   | 0.149   | 0.089   | **0.061**| **0.129**| 0.130   |
+| Qwen2.5-Omni-7B   | **0.328**| **0.162**| **0.090**| 0.055   | 0.127    | **0.225**|
+
+### Audio Reasoning
+
+#### Dataset: MMAU
+
+| Model            | Sound  | Music  | Speech | Avg    |
+|------------------|--------|--------|--------|--------|
+| Gemini-Pro-V1.5  | 56.75  | 49.40  | 58.55  | 54.90  |
+| Qwen2-Audio      | 54.95  | 50.98  | 42.04  | 49.20  |
+| Qwen2.5-Omni-7B  | **67.87** | **69.16** | **59.76** | **65.60** |
+
+### Voice Chatting
+
+#### Dataset: VoiceBench
+
+
+| Model                              | AlpacaEval | CommonEval | SD-QA   | MMSU   |
+|------------------------------------|------------|------------|---------|--------|
+| Ultravox-v0.4.1-LLaMA-3.1-8B       | **4.55**   | 3.90       | 53.35   | 47.17  |
+| MERaLiON                           | 4.50       | 3.77       | 55.06   | 34.95  |
+| Megrez-3B-Omni                     | 3.50       | 2.95       | 25.95   | 27.03  |
+| Lyra-Base                          | 3.85       | 3.50       | 38.25   | 49.74  |
+| MiniCPM-o                          | 4.42       | **4.15**   | 50.72   | 54.78  |
+| Baichuan-Omni-1.5                  | 4.50       | 4.05       | 43.40   | 57.25  |
+| Qwen2-Audio                        | 3.74       | 3.43       | 35.71   | 35.72  |
+| Qwen2.5-Omni-7B                    | 4.49       | 3.93       | **55.71**| **61.32**|
+
+
+| Model                              | OpenBookQA | IFEval  | AdvBench | Avg    |
+|------------------------------------|------------|---------|----------|--------|
+| Ultravox-v0.4.1-LLaMA-3.1-8B       | 65.27      | **66.88** | 98.46   | 71.45  |
+| MERaLiON                           | 27.23      | 62.93   | 94.81    | 62.91  |
+| Megrez-3B-Omni                     | 28.35      | 25.71   | 87.69    | 46.25  |
+| Lyra-Base                          | 72.75      | 36.28   | 59.62    | 57.66  |
+| MiniCPM-o                          | 78.02      | 49.25   | 97.69    | 71.69  |
+| Baichuan-Omni-1.5                  | 74.51      | 54.54   | 97.31    | 71.14  |
+| Qwen2-Audio                        | 49.45      | 26.33   | 96.73    | 55.35  |
+| Qwen2.5-Omni-7B                    | **81.10**  | 52.87   | **99.42**| **74.12**|
 
```

Comment:
Note that Table 4 in the paper is missing from markdown (should I add it?)
<img width="528" alt="Screenshot 2025-04-08 at 23 29 31" src="https://github.com/user-attachments/assets/762faf61-97f6-44a1-9691-05a8a12c6c17" />